datashuttle/datashuttle.py

```diff
@@ -1451,3 +1456,16 @@ def _update_settings_with_new_canonical_keys(self, settings: Dict):
         for key in ["overwrite_existing_files", "dry_run"]:
             if key not in settings["tui"]:
                 settings["tui"][key] = canonical_tui_configs["tui"][key]
+
+    def _check_top_level_folder(self, top_level_folder):
+        """
+        Raise an error if `top_level_folder` not correct.
```

Comment:
Needs doube backticks to rended as monospace in sphinx. As it is now, the result will be italicised ( I think). Not that it matters much, since this is a private functions.

```suggestion
Raise an error if ``top_level_folder`` not correct.
````